ops/roles/cgus/tasks/database.yml

```diff
@@ -0,0 +1,62 @@
+- name: Get information about {{ name }}
+  git:
+    repo: "{{ repository }}"
+    dest: "/home/{{ ansible_user }}/{{ directory }}"
+    clone: no
+    update: no
+    accept_hostkey: yes
+    key_file: "/home/{{ ansible_user }}/.ssh/cgus-bot-key"
+  register: git_dir
+  tags:
+    - restart
+    - start
+    - update
+
+- name: Remove index.lock in {{ name }}
+  file:
+    path: "/home/{{ ansible_user }}/{{ directory }}/.git/index.lock"
+    state: absent
+  when: git_dir.before is defined and git_dir.before
+  tags:
+    - restart
+    - start
+    - update
+
+- name: Clone {{ name }}
+  git:
+    repo: "{{ repository }}"
+    dest: "/home/{{ ansible_user }}/{{ directory }}"
+    depth: 1
+    accept_hostkey: yes
+    key_file: "/home/{{ ansible_user }}/.ssh/cgus-bot-key"
+  when: git_dir.before is defined and not git_dir.before
```

Comment:
I don't understand any of these conditionals, and the Ansible documentation really doesn't help me at all. We need to document this. Might be easier to talk 🗣️